### PR TITLE
KAA-1371: Your first Kaa application (updated with CTL info).

### PR DESCRIPTION
**Pull request by hunch7**

### CODE REVIEW

File: doc/Programming-guide/Your-first-Kaa-application/index.md [hunk 1]
```diff
@@ -767,43 +830,53 @@ Sampled temperature: 31
...
-    of the application the Administration UI.
+1. Copy the application token from your new application page.
 
     ![Generate SDK](attach/app_token.png)
-1. Login to your machine running Kaa server (in case you are using sandbox,
-    default username and password are `kaa`/`kaa`).
-1. Start MongoDB shell
+
+2. Log in to your machine running Kaa server or to your Sandbox.
+For the Sandbox, use the default **kaa** username and **kaa123** password.
```
Comment (ikulikov) [inline doc/Programming-guide/Your-first-Kaa-application/index.md]: The way how user should log in to the Sandbox is not clear here. User should login to the Sandbox VM using ssh with kaa/kaa credentials.

**Review by ikulikov — CHANGES_REQUESTED** [review]

File: doc/Programming-guide/Your-first-Kaa-application/index.md (hunk 1 above)
Comment (hunch7) [reply doc/Programming-guide/Your-first-Kaa-application/index.md]: Fixed.

File: doc/Programming-guide/Your-first-Kaa-application/index.md [hunk 2]
```diff
@@ -10,147 +10,161 @@ sort_idx: 10
...
+					"by_default": "1"
+				}
+			]
+		}
+		
+3. Use the **Upload from file** section to [create CT schemas]({{root_url}}/Programming-guide/Key-platform-features/Common-Type-Library/#create-a-new-ct) of Application scope from the data-schema.json and configuration-schema.json files.
+		
+4. On the Sandbox main page, click **Administration** and log in as a tenant developer using default **devuser** username and **devuser123** password.
+
+5. Click the **Applications** arrow to unfold the list and click the arrow of the application you created in [Adding an application](#adding-an-application), then click **Schemas** > **Log** and click the **Add schema** button.
```
Comment (Polinaa) [inline doc/Programming-guide/Your-first-Kaa-application/index.md:91]: Bad link, should be **#add-application**

File: doc/Programming-guide/Your-first-Kaa-application/index.md [hunk 3]
```diff
@@ -10,147 +10,161 @@ sort_idx: 10
...
-After picking correct versions for schemas used in SDK click **Add**.
+3. On the **Add SDK profile** page, fill in the fields as below.
 
-    ![Configure SDK](attach/configure_sdk.png)
-1. Now the SDK is configured and ready to be generated. Click **Generate SDK** for corresponding SDK profile.
-In the **Generate SDK** window select the target platform for your SDK and click Generate SDK.
+	![Configure SDK](attach/configure_sdk.png)
 
-    ![Generate SDK](attach/generate_configured_sdk.png)
+	>**NOTE:** Version numbers in the screenshot correspond to the version numbers obtained in [Creating schemas](#creating-schemas).
```
Comment (Polinaa) [inline doc/Programming-guide/Your-first-Kaa-application/index.md:140]: Bad link, should be **#create-schemas**

File: doc/Programming-guide/Your-first-Kaa-application/index.md [hunk 4]
```diff
@@ -10,147 +10,161 @@ sort_idx: 10
...
+
+		{
+			"type": "record",
+			"name": "Configuration",
+			"namespace": "org.kaaproject.kaa.schema.sample",
+			"fields": [
+				{
+					"name": "samplePeriod",
+					"type": "int",
+					"by_default": "1"
```
Comment (Polinaa) [inline doc/Programming-guide/Your-first-Kaa-application/index.md:82]: "by_default": **1**

File: doc/Programming-guide/Your-first-Kaa-application/index.md [hunk 5]
```diff
@@ -10,147 +10,161 @@ sort_idx: 10
...
+
+	![Data collection schema](attach/new_data_schema2.png)
 
-    `by_default` field defines the default value for the sampling period, which is in our example
-    is set to 1 second.
-1. Create CTL schemas of Application scope using **Upload from file** section with previously created files `data-schema.json` and `configuration-schema.json` : [how to create CTL schema]({{root_url}}/Programming-guide/Key-platform-features/Common-Type-Library/#create-a-new-ct).
-1. Open the [admin UI]({{root_url}}Administration-guide/Tenants-and-applications-management) in your browser and log in as a tenant developer (default user/password in the sandbox: `devuser`/`devuser123`; use the correct credentials if you changed the default).
-1. Open the relevant **Log schemas** window (**Applications => My First Kaa Application => Schemas => Log**) and click **Add schema**.
-1. In the **Add log schema** window, enter the name and description of the new data collection schema and select CTL created before (from file `data-schema.json`).
+7. Scroll down and use the **Upload from file** function to find the previously created data-schema.json file and click **Upload**.
```
Comment (Polinaa) [inline doc/Programming-guide/Your-first-Kaa-application/index.md:97]: Unnecessary step as CT was created and used before (_3_ and _6_ steps).

File: doc/Programming-guide/Your-first-Kaa-application/index.md (hunk 4 above)
Comment (hunch7) [reply doc/Programming-guide/Your-first-Kaa-application/index.md:82]: Fixed.

File: doc/Programming-guide/Your-first-Kaa-application/index.md (hunk 2 above)
Comment (hunch7) [reply doc/Programming-guide/Your-first-Kaa-application/index.md:91]: Fixed.

File: doc/Programming-guide/Your-first-Kaa-application/index.md (hunk 5 above)
Comment (hunch7) [reply doc/Programming-guide/Your-first-Kaa-application/index.md:97]: Fixed.

File: doc/Programming-guide/Your-first-Kaa-application/index.md (hunk 3 above)
Comment (hunch7) [reply doc/Programming-guide/Your-first-Kaa-application/index.md:140]: Fixed.

**Comment by hunch7** [comment]
Latest fixes upon QA review: https://github.com/kaaproject/kaa/pull/1114
@ikulikov , please merge.

**Pull request merged**